packages/app/provider/index.tsx

```diff
@@ -98,7 +57,7 @@ export const setTheme = async (
 ) => {
   try {
     setActiveTheme(theme);
```

Comment:
is continuing to set this manually here (and in `clearTheme`) just a perf optimization? I'd expect the store action to trigger a change in `useSyncedAppTheme` itself via the `useThemeSettings` hook